app/src/split/bluetooth/peripheral.c

```diff
@@ -57,10 +59,12 @@ static int start_advertising(bool low_duty) {
     bt_foreach_bond(BT_ID_DEFAULT, each_bond, &central_addr);
 
     if (bt_addr_le_cmp(&central_addr, BT_ADDR_LE_NONE) != 0) {
+        is_bonded = true;
         struct bt_le_adv_param adv_param = low_duty ? *BT_LE_ADV_CONN_DIR_LOW_DUTY(&central_addr)
                                                     : *BT_LE_ADV_CONN_DIR(&central_addr);
         return bt_le_adv_start(&adv_param, NULL, 0, NULL, 0);
     } else {
+        is_bonded = false;
```

Comment:
I don't see anywhere this `is_bonded` gets updated after the central actually bonds with the peripheral?